src/injected/visualization/svg-shape-factory.ts

```diff
@@ -84,6 +84,7 @@ export class SVGShapeFactory {
         center: Point,
         textConfig: TextConfiguration,
         tabOrder: number,
+        innerText?: string,
```

Comment:
I think a cleaner approach here might be to simply pass in the inner text as a parameter here. The SVG drawer can dictate what that text is (instead of having to null check two separate properties here).

Reply:
Do you mean moving the checks to the svg-drawer?

Reply:
Yep!